woocommerce/woocommerce
# Added wp-textdomain to fix Package's text domains

### All Submissions:






### Changes proposed in this Pull Request:



Currently our script replaces all instances of `woo-gutenberg-products-block` and `woocommerce-rest-api`, what isn't a perfect solution since those slugs can be found or used in other contexts that doesn't involves text domains.

[`wp-textdomain`](https://www.npmjs.com/package/wp-textdomain) helps us to update/fix only text domains, also helps fixing for any package included, so no updates are required if we include new packages (like WooCommerce Admin or Action Scheduler #25404).

Currently it output a log showing every translatable string that got updated, if it's too annoying we can disable it.

### How to test the changes in this Pull Request:

1. Checkout this branch
2. Remove `packages/woocommerce-blocks` and `packages/woocommerce-rest-api` directories.
3. Run `composer install`, all packages will get included again and the new script will run fixing all text domains.
4. Check for the output log, and also for files in any package to check if text domains got fixed.

### Other information:
